client/templates/home.html

```diff
@@ -1,12 +1,32 @@
 <template name="home">
 
-  {{#if currentUser}}
-    {{> meetingSeriesList}}
+  {{#if sendVerificationEmailEnabled}}
+      {{#unless currentUser.emails.[0].verified}}
+          <p class="alert alert-warning">You need to verify your email address before.
```

Comment:
To be honest: I did not think of these security scenarios.
I thought: if a user is not verified he is not able to log in.
So, I would suggest we leave this aspect out of this story and put it to a story for next sprint.